srcpkgs/rpi-kernel/template

```diff
@@ -27,8 +27,8 @@ noverifyrdeps=yes
 noshlibprovides=yes
 
 # RPi, RPi2, RPi3
-only_for_archs="armv6l armv6l-musl armv7l armv7l-musl aarch64 aarch64-musl"
-hostmakedepends="perl kmod uboot-mkimage libressl-devel bc"
+archs="armv6l armv6l-musl armv7l armv7l-musl aarch64 aarch64-musl"
```

Comment:
can be replaced by globs

armv6l* armv7l* aarch64*

Reply:
Updated to use globs.